pipelines/tasks/Attach10xBarcodes.wdl

```diff
@@ -0,0 +1,30 @@
+
+task Attach10xBarcodes {
+  File r1  # forward fastq file; contains umi, cell barcode
+  File i7  # index fastq file; contains sample barcode
+  File u2  # reverse bam file; contains alignable genomic information
+
+  # estimate disk requirements
+  Float bam_size = size(u2, "G")
+  Float fastq_size = size(r1, "G") + size(i7, "G")
+  Int estimated_required_disk = ceil(fastq_size + bam_size * 2.2)
+
+  command {
+    Attach10xBarcodes \
+      --r1 ${r1} \
```

Comment:
You generally want to double-quote all of these things. There's some discussion [here](http://mywiki.wooledge.org/Quotes).